pygeoapi/provider/filesystem.py

```diff
@@ -257,31 +259,35 @@ def _describe_file(filepath):
     except rasterio.errors.RasterioIOError:
         LOGGER.debug('Testing vector data detection')
         d = fiona.open(filepath)
-
+        tcrs = CRS.from_epsg(4326)
```

Comment:
Should/can we test whether the data's CRS is 4326, and if not, then reproject?

Reply:
Or projection nill, also is quite common